### PR TITLE
Add ProductMenu class

### PR DESCRIPTION
**Pull request by bertrandong**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteMenuCommand.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.order.Product;
+
+/**
+ * Deletes a product identified using it's displayed index from the menu.
+ */
+public class DeleteMenuCommand extends DeleteCommand {
```
Comment (chonghaoen) [inline src/main/java/seedu/address/logic/commands/DeleteMenuCommand.java:17]: good implementation

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java [hunk 2]
```diff
@@ -17,12 +19,25 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
...
+        requireNonNull(args);
+
+        // See if still needed
+        //try {
+        //    Index index = ParserUtil.parseIndex(args);
+        //    return new DeleteCommand(index);
+        //} catch (ParseException pe) {
+        //    throw new ParseException(
+        //            String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE), pe);
+        //}
```
Comment (chonghaoen) [inline src/main/java/seedu/address/logic/parser/DeleteCommandParser.java:31]: likely not

**Review by chonghaoen — COMMENTED** [review]
LGTM :)

File: src/main/java/seedu/address/logic/commands/AddMenuCommand.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (model.hasProduct(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_PRODUCT);
+        }
+
+        model.addProduct(toAdd);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, Messages.format(toAdd)));
+    }
```
Comment (nigel27022001) [inline src/main/java/seedu/address/logic/commands/AddMenuCommand.java:50]: Good Implementation! Good job

File: src/main/java/seedu/address/logic/commands/DeleteCustomerCommand.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        model.deletePerson(personToDelete);
+        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, Messages.format(personToDelete)));
+    }
+
```
Comment (nigel27022001) [inline src/main/java/seedu/address/logic/commands/DeleteCustomerCommand.java:46]: Good use of lastshownlist

**Pull request merged**